feature/networkinstance/networkinstance.go

```diff
@@ -49,9 +51,24 @@ func (ni *NetworkInstance) validate() error {
 	if ni.oc.GetType() == oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET {
 		return errors.New("NetworkInstance type is unset")
 	}
+	if ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic("prefix").GetOrCreateNextHop("0").NextHop == oc.UnionString("") {
+		return errors.New("NetworkInstance nexthop is unset")
+	}
 	return ni.oc.Validate()
 }
 
+// WithStaticRoute sets the prefix value for static route.
+func (ni *NetworkInstance) WithStaticRoute(prefix string, nextHops []string) *NetworkInstance {
+	ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic("prefix").Prefix = ygot.String(prefix)
```

Comment:
I believe we should use the actual prefix instead of "prefix".

GetOrCreateStatic(prefix)

Yeah, it would be a repeat but that is how it is done for config so better to be consistent. Also we need it to be unique so using the actual prefix will get you that.

Reply:
Also shouldn't this be GetOrCreateProtocol(...).GetOrCreateStaticRoutes().GetOrCreateStatic(...) ? Per the OC path, I would have presumed this would be the case but it is compiling for you so I am surprised.

Reply:
Done!

Reply:
Actually https://source.corp.google.com/piper///depot/google3/blaze-out/genfiles/ops/netops/openconfig/yang/structs-2.go;rcl=HEAD;l=389 suggests that GetOrCreateStatic is the correct path and using GetOrCreateStaticRoutes() gives compilation errors.